src/linux-dev-certs/CertificateManager.cs

```diff
@@ -188,14 +206,28 @@ private List<ICertificateStore> FindAdditionaCertificateStores()
             FindFirefoxCertificateStores(firefoxSnapUserDirectory, stores);
         }
 
+        string firefoxUserDirectory = Path.Combine(home, ".mozilla/firefox");
+        if (OSFlavor.IsGentooLike && Directory.Exists(firefoxUserDirectory))
+        {
+            FindFirefoxCertificateStores(firefoxUserDirectory, stores);
+        }
+
+        string librewolfUserDirectory = Path.Combine(home, ".librewolf");
+        if (OSFlavor.IsGentooLike && Directory.Exists(librewolfUserDirectory))
+        {
+            FindLibrewolfCertificateStores(librewolfUserDirectory, stores);
```

Comment:
The `FindLibrewolfCertificateStores` method seems to do the same thing as `FindFirefoxCertificateStores`, can you call `FindFirefoxCertificateStores` here and remove `FindLibrewolfCertificateStores`?

Reply:
This is Firefox under the hood. I don't want to duplicate the code for rebranding.

You can add a `browserName` variable to the existing `FindFirefoxCertificateStores` method.
```cs
private static void FindFirefoxCertificateStores(string firefoxUserDirectory, List<ICertificateStore> stores, string browserName = "Firefox")
```
and use `{browserName}` instead of `Firefox` here:
https://github.com/tmds/linux-dev-certs/blob/e355e7bfd4b4cb9f0bf49fa6c57284bcbca4053c/src/linux-dev-certs/CertificateManager.firefox.cs#L40

Reply:
fixed in [11e5cec](https://github.com/tmds/linux-dev-certs/pull/22/commits/11e5cec71f7001f0dc382e16f04bedc799238f32)